web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -980,9 +986,20 @@ $.extend(fixmystreet.set_up, {
   },
 
   response_templates: function() {
+      // If the user has manually edited the contents of an update field,
+      // mark it as dirty so it doesn't get clobbered if we select another
+      // response template. If the field is empty, it's not considered dirty.
+      $('.js-template-name').each(function() {
+          var $input = $('#' + $(this).data('for'));
+          $input.change(function() { $(this).data('dirty', !/^\s*$/.test($(this).val())); });
+      });
+
       $('.js-template-name').change(function() {
           var $this = $(this);
-          $('#' + $this.data('for')).val($this.val());
+          var $input = $('#' + $this.data('for'));
+          if (!$input.data('dirty')) {
```

Comment:
Perhaps here could drop the `dirty` flag and use `if ($input[0].defaultValue != $input.val())` – this wouldn't have the 'empty field' functionality but otherwise I think is the same.